exchanges/bitmex/bitmex.go

```diff
@@ -887,17 +876,20 @@ func (b *Bitmex) SendAuthenticatedHTTPRequest(ep exchange.URL, verb, path string
 
 	ctx, cancel := context.WithDeadline(context.Background(), expires)
 	defer cancel()
-	err = b.SendPayload(ctx, &request.Item{
+	fullPath := endpoint + path
```

Comment:
This function also uses timestamp based signature and headers. I would advise inlining. Bittrex has rate limits and can be prone to expiry